tensorrt_llm/_torch/auto_deploy/transform/library/multi_stream_attn.py

```
@@ -0,0 +1,219 @@
+        # Ensure aux stream and events are set up for the current device.
+        cuda_stream_manager.add_device(torch.cuda.current_device())
+
+        gm, num_matches = _execute_kv_proj_in_aux_stream(gm)
+
+        info = TransformInfo(
+            skipped=False,
+            num_matches=num_matches,
+            is_clean=num_matches == 0,
+            has_valid_shapes=num_matches == 0,
+        )
+        with open("/tmp/after_multi_stream_mla_attn.txt", "w") as f:
```

Ruff (0.15.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/after_multi_stream_mla_attn.txt"

(S108)